### PR TITLE
Customize theme to have better links in sidebar

### LINKED ISSUE #61: Add link to GitHub issues from the RTD site
tljh.jupyter.org does not have a link to this repository. It should!

### PR DESCRIPTION
**Pull request by yuvipanda**
Fixes #61

### CODE REVIEW

**Comment by yuvipanda** [comment]
@choldgraf this seems to have 0 effect when I run 'make html' locally :( any idea what could be causing that?

File: docs/conf.py [hunk 1]
```diff
@@ -36,3 +36,14 @@ def setup(app):
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+
+# Custom Alabaster theme options
+html_theme_options = {
+    'description': 'A simple JupyterHub distribution for 1-100 users',
+    'github_user': 'jupyterhub',
+    'github_repo': 'the-littlest-jupyterhub',
+    'github_button': 'true',
```
Comment (Carreau) [inline docs/conf.py]: Boolean not string

File: docs/conf.py [hunk 2]
```diff
@@ -36,3 +36,14 @@ def setup(app):
...
+    'extra_nav_links': {
+        'Documentation confusing? File an issue!': 'https://github.com/jupyterhub/the-littlest-jupyterhub/issues'
```
Comment (Carreau) [inline docs/conf.py]: Needs something like

```
html_sidebars = {
    '**': [
        'about.html',
        'navigation.html',
        'relations.html',
        'searchbox.html',
        'donate.html',
    ]
}
```

To be set, but fuck up the layout and put a link to all the pages.

Comment (Carreau) [reply docs/conf.py]: screen shot 2018-07-29 at 11 51 23

**Comment by yuvipanda** [comment]
@Carreau this is awesome! <3 tyvm.

I wonder if there's a way to remove the 'all the links in the page' part of the navigation.

**Comment by yuvipanda** [comment]
@choldgraf nope, @Carreau's idea fixed it!

**Pull request merged**